pages/explorer.py

```diff
@@ -136,13 +137,13 @@ def section_one():
                                 className="mb-2",
                                 n_clicks=0,
                             ),
-                            html.Div(
+                            dmc.Box(
```

Comment:
here most likely we want to remove this div all together not just use a dmc.Box instead. We should simplify the code as much as possible and remove the classes. We should only be using mantine and apply styles like padding and margins, if needed in line